edge/pkg/devicetwin/dtmanager/device.go

```diff
@@ -113,7 +118,7 @@ func dealDeviceStateUpdate(context *dtcontext.DTContext, resource string, msg in
 		dtcommon.CommModule,
 		context.BuildModelMessage(modules.BusGroup, "", topic, messagepkg.OperationPublish, payload))
 
-	msgResource := "device/" + device.ID + "/state"
+	msgResource := "device/" + device.ID + dtcommon.DeviceETStateUpdateSuffix
```

Comment:
This is a bug?

Reply:
Actually here updating '/state' to '/state/update', is it intend?

Reply:
the topic which used by mapper for updateing state of device is ```"$hw/events/device/+/state/update"``` and I think it's better to use public constant rather than local too